app/controllers/application_controller.rb

```diff
@@ -22,6 +22,26 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
+  #By default only show breadcrumbs to logged-in users.
+  def show_breadcrumbs
+    current_ability.current_user.logged_in?
+  end
+  helper_method :show_breadcrumbs
```

Comment:
call it `show_breadcrumbs?`, question mark at the end is a nice convention for boolean methods, as seen in existing `logged_in?`

Reply:
Cool, will do!